.github/workflows/tests.yml

```diff
@@ -41,6 +41,12 @@ jobs:
         run: git checkout --force ci-branch
       - name: "Log commits"
         run: git log -3
+      # --- Cache ---
+      - name: "Configure build cache"
+        uses: actions/cache@v2
+        with:
+          path: /usr/local/faasm/object
+          key: ${{ runner.os }}-cache
```

Comment:
We'll see how consistent the architecture of the Github runners is. This may have to become specific to the underlying processor